README.md

```diff
@@ -138,3 +138,45 @@ final class SomeClass implements MyInterface
 }
 ```
 :+1:
+
+### Check calling method in object method
+Checks if some method is not used in disabled context - specific method of object.
+```neon
+services:
+    disableCallMethodInObjectMethodRule:
+        factory: Efabrica\PHPStanRules\Rule\General\DisableCallMethodInObjectMethodRule('pageConfiguration', 'setRequired')
```

Comment:
I miss some other context: context class, context method, caller class, called method. You have just context method and called method. I am not sure if it is enough